recipes/ozo/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import check_min_cppstd, Version
+import os
+import glob
+
+
+class OzoConan(ConanFile):
+    name = "ozo"
+    description = "C++ header-only library for asynchronous access to PostgreSQL databases using ASIO"
+    topics = ("ozo", "yandex", "postgres", "postgresql", "cpp17", "database", "db", "asio")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/yandex/ozo"
+    license = "PostgreSQL"
+
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
Only compiler is used
```suggestion
    settings = "compiler"
```

Reply:
Maybe I have a misunderstanding regarding the conan architecture, but isn't at least `os` necessary to reject windows?

Reply:
You are absolutely right, I must have missed that